tests/test_smoke.py

```diff
@@ -3108,6 +3141,44 @@ def test_multiple_buckets_creation_and_deletion(
         ]
         assert all([item not in out for item in storage_obj_name])
 
+    @pytest.mark.parametrize('store_type', [
+        storage_lib.StoreType.S3, storage_lib.StoreType.GCS,
+        pytest.param(storage_lib.StoreType.IBM, marks=pytest.mark.ibm),
+        pytest.param(storage_lib.StoreType.R2, marks=pytest.mark.cloudflare)
+    ])
+    def test_upload_source_with_spaces(self, store_type,
+                                       tmp_multiple_custom_source_storage_obj):
+        # Creates four buckets with specified local source paths and deletes them
```

Comment:
```suggestion
        # Creates four buckets with specified local source paths
```
We don't need to delete the storage objects from this test function.